### PR TITLE
Move 6.0.0 rc.2 changes

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Merge the following PRs to 6.0.0-rc.2:
#5623  Note: merge conflicts had to be resolved because we are specifying release tag in most `start-psbuild` statements in master:
*	tools/appveyor.psm1
*	tools/travis.ps1

#5631

### CODE REVIEW

**Review by daxian-dbw — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 - Update the contribution guideline to note that updating the changelog is required. (#5586)
+- Remove Pester as a module include with the PowerShell Packages.
+  In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Should we have this as a sub bullet point to make it clear it's part of the previous sentence.  I think typically we've had changes on a single line, but would not want long lines either.

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rkeithhill) [reply CHANGELOG.md]: Shouldn't `Start-PSBootstrap` check for Pester and just install it (CurrentUser scope) if it isn't on the user's system?

Comment (TravisEz13) [reply CHANGELOG.md]: Why is not a single bullet point adequate?

Comment (TravisEz13) [reply CHANGELOG.md]: `start-psbootstrap` is for the development environment, this is referring to the product.

**Review by SteveL-MSFT — APPROVED** [review]

**Pull request merged**